servers/quarkus-common/src/main/java/org/projectnessie/quarkus/config/QuarkusStoreConfig.java

```diff
@@ -21,15 +21,25 @@
 import io.smallrye.config.WithDefault;
 import io.smallrye.config.WithName;
 import java.time.Duration;
+import java.util.List;
 import java.util.Optional;
 import java.util.OptionalDouble;
 import java.util.OptionalInt;
 import org.projectnessie.versioned.storage.common.config.StoreConfig;
 
 @StaticInitSafe
-@ConfigMapping(prefix = "nessie.version.store.persist")
+@ConfigMapping(prefix = QuarkusStoreConfig.NESSIE_VERSION_STORE_PERSIST)
 public interface QuarkusStoreConfig extends StoreConfig {
 
+  String NESSIE_VERSION_STORE_PERSIST = "nessie.version.store.persist";
+  String CONFIG_CACHE_INVALIDATIONS_VALID_TOKENS = "cache-invalidations.valid-tokens";
```

Comment:
Nit: I don't get why `cache-invalidations` in the plural. The corresponding method names are in the singular, e.g. `cacheInvalidationServiceNames()`